jdaviz/app.vue

```diff
@@ -120,7 +120,7 @@
                       </j-tooltip>
                     </v-expansion-panel-header>
                     <v-expansion-panel-content style="margin-left: -12px; margin-right: -12px;">
-                      <jupyter-widget :widget="trayItem.widget"></jupyter-widget>
+                      <jupyter-widget :widget="trayItem.widget" v-if="state.tray_items_open.includes(index)"></jupyter-widget>
```

Comment:
this is needed so that the same logic works for plugins in the tray (otherwise the plugin element when collapsed will continue sending pings)